src/utils.ts

```diff
@@ -155,6 +155,9 @@ export function offsetToLineAndColumn(string: string, offset: number) {
   let line = 0;
   let seen = 0;
   while (true) {
+    if (line >= lines.length) {
+      throw new Error(`offset ${offset} exceeded string ${JSON.stringify(string)}`);
```

Comment:
You sure you want to put the whole string here? That could be really long. Why not just put the maximum offset or refer to the lines or something?

Reply:
This is basically an assertion failure. The point of the error message is to have the whole string so I can see what I did wrong. I don't expect it to get surfaced to users.